### PR TITLE
added author info

### PR DESCRIPTION
**Pull request by nrosed**
added in the author info! I added a note, too :)

I wasn't exactly sure how to add it in, but I added it as another author in the metadata.yaml. I think it looks ok, but let me know if there is a better way.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -1,26 +1,37 @@
 ---
 title: "Computational audits combat disparities in recognition"
 keywords:
-  - markdown
-  - publishing
-  - manubot
+  - opinion
+  - scientific equity
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
  - equity
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
+  - github: cgreene
+    name: Casey S. Greene
+    orcid: 0000-0001-8713-9213
+    twitter: GreeneScientist
+    email: casey.s.greene@cuanschutz.edu
+    affiliations:
+      - University of Colorado School of Medicine, Aurora, Colorado, United States of America; Center for Health AI
+    funders:
+      - The Gordon and Betty Moore Foundation (GBMF 4552)
+  - name: \* *denotes co-first authorship. Order determined by d20.*
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
  - name: \* *denotes co-first authorship. Order determined by simultaneous rolls of d20s over a video call.*
```

**Review by cgreene — APPROVED** [review]
I made a few comments - let's see how it renders 🤞

**Comment by nrosed** [comment]
> Did we miss the deadline on submitting this?

It's not a hard deadline, just our proposed deadline we discussed. @cgreene please correct me if I'm wrong

**Review by arielah — APPROVED** [review]

**Comment by cgreene** [comment]
I agree I don't think the deadline was hard but I think this sets us up to submit - is that right? If so I will try to get it out tomorrow

**Comment by arielah** [comment]
> I agree I don't think the deadline was hard but I think this sets us up to submit - is that right? If so I will try to get it out tomorrow

Think so! Thanks Casey

**Pull request merged**